hnn_core/pyramidal.py

```diff
@@ -848,12 +848,12 @@ def parconnect(self, gid, gid_dict, pos_dict, p):
         postsyns = [self.apicaloblique_ampa, self.basal2_ampa,
                     self.basal3_ampa]
         self._connect(gid, gid_dict, pos_dict, p,
-                      'L5_pyramidal', 'L5Pyr', lamtha=3., receptor='ampa',
+                      'L5Pyr', 'L5Pyr', lamtha=3., receptor='ampa',
```

Comment:
the issue now is that you pass 'L5Pyr' twice in `_connect` which is not necessary. One of the arguments to the method can be removed.

But we'll need to wait for #146 to be merged first to avoid a rebase mess